test/arrays/ferguson/return-array-as-int.chpl

```diff
@@ -0,0 +1,12 @@
+// From issue #8392
+
+record NamedMatrix {
+  var rowNames:[1..10] int;
+}
+
+proc NamedMatrix.setRowNames(rn:[]):int {
```

Comment:
Is the argument declaration important?

Reply:
I doubt it - I just copied this from an issue though, it's really a form of the bug report.

Reply:
I'm okay with leaving it, then